### PR TITLE
Blocks support

### PR DESCRIPTION
**Pull request by chesstrian**
Support for layout blocks: https://api.slack.com/messaging/composing/layouts

### CODE REVIEW

**Comment by chesstrian** [comment]
Hi @lamby, it'd be great to have this feature in the next version, please let me know if this makes sense for you or something additional is required.

```python
slack_message('app/template.slack', blocks=[{
    'type': 'divider'
}])
```

File: django_slack/api.py [hunk 1]
```diff
@@ -132,6 +132,9 @@ def slack_message(template, context=None, attachments=None, fail_silently=None,
 
         if 'attachments' in data:
             data['attachments'] = json.dumps(data['attachments'])
+        
+        if 'blocks' in kwargs:
```
Comment (lamby) [inline django_slack/api.py]: Not sure this is right; should one be adding something to `PARAMS` earlier in the method rather than a special-case here?

**Comment by lamby** [comment]
@chesstrian Looks great. Can you just address these concerns and we can merge it ASAP.

**Comment by chesstrian** [comment]
@lamby, changes done

File: django_slack/api.py [hunk 2]
```diff
@@ -132,6 +137,9 @@ def slack_message(template, context=None, attachments=None, fail_silently=None,
 
         if 'attachments' in data:
             data['attachments'] = json.dumps(data['attachments'])
+        
+        if 'blocks' in data:
```
Comment (lamby) [inline django_slack/api.py]: Neat, now we can have a nice loop over a `('attachments', 'blocks')` tuple so do avoid some duplicated code here.

File: django_slack/api.py [hunk 3]
```diff
@@ -135,11 +135,9 @@ def slack_message(template, context=None, attachments=None, blocks=None, fail_si
                     "specified".format(k),
                 )
 
-        if 'attachments' in data:
-            data['attachments'] = json.dumps(data['attachments'])
-        
-        if 'blocks' in data:
-            data['blocks'] = json.dumps(data['blocks'])
+        for arg in ('attachments', 'blocks',):
```
Comment (lamby) [inline django_slack/api.py]: Can I be super pedantic and ask you to use `x` as the anonymous variable and drop the trailing `,` ? :)

**Comment by lamby** [comment]
Awesome, this has been released in `5.14.0`. :)

**Comment by chesstrian** [comment]
Great, thanks a lot :slightly_smiling_face:

**Pull request merged**